rust/rsc/src/rsc/blob_store_service.rs

```diff
@@ -51,6 +54,7 @@ pub async fn fetch_unreferenced_blobs(
                 UNION SELECT stdout_blob_id FROM job
                 UNION SELECT stderr_blob_id FROM job
             )
+            LIMIT 16000
```

Comment:
It seems like we should also limit the query in `delete_blobs_by_ids` in case we have a very long list of ids.